### PR TITLE
Created figures with calculated area

### PR DESCRIPTION
**Pull request by Evgenb000**

### CODE REVIEW

**Review by VenomMB — APPROVED** [review]
GJ!!!

File: src/figuresClasses.ts [hunk 1]
```diff
@@ -1,19 +1,102 @@
...
+  readonly shape = 'triangle';
 
-}
+  readonly color: Color;
+
+  readonly a: number;
+
+  readonly b: number;
+
+  readonly c: number;
```
Comment (liliia-livandovskaya) [inline src/figuresClasses.ts:19]:
```suggestion
  readonly color: Color;
  readonly a: number;
  readonly b: number;
  readonly c: number;
```

File: src/figuresClasses.ts [hunk 2]
```diff
@@ -1,19 +1,102 @@
...
+    }
+
+    this.color = color;
+    this.a = a;
+    this.b = b;
+    this.c = c;
+  }
+
+  getArea(): number {
+    const s = (this.a + this.b + this.c) / 2;
```
Comment (liliia-livandovskaya) [inline src/figuresClasses.ts:37]:
```suggestion
    const semiPerimeter = (this.a + this.b + this.c) / 2;
```

File: src/figuresClasses.ts (hunk 1 above)
Comment (liliia-livandovskaya) [reply src/figuresClasses.ts:19]: you can use `public` instead of `readonly`

**Review by liliia-livandovskaya — APPROVED** [review]

**Pull request closed without merging**